third_party/3/lxml/etree.pyi

```diff
@@ -17,23 +17,67 @@ _DictAnyStr = Union[Dict[str, str], Dict[bytes, bytes]]
 _Dict_Tuple2AnyStr_Any = Union[Dict[Tuple[str, str], Any], Tuple[bytes, bytes], Any]
 
 
-class ElementChildIterator(Iterator['_Element']):
+class _ElementMatchIterator(Iterator['_Element']):
+    def __iter__(self) -> '_ElementMatchIterator': ...
```

Comment:
All of these overridden `__iter__`/`__next__` methods seem to be redundant. I think you can just write most of these classes like `class _ElementMatchIterator(Iterator[_Element]): ...`. `__iter__` and `__next__` will be inherited correctly. In that case, leave no blank space between the classes.